extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -459,10 +461,15 @@ class MembershipsButtonEdit extends Component {
 								notices={ notices }
 							>
 								<div className="components-placeholder__instructions">
-									<p>{ __( 'Select payment plan:', 'jetpack' ) }</p>
+									<p>
+										{ __(
+											'To use this block, select a previously created payment plan.',
+											'jetpack'
+										) }
+									</p>
 									{ this.renderMembershipAmounts() }
-									<p>{ __( 'Or add another Recurring Payments plan:', 'jetpack' ) }</p>
-									{ this.renderAddMembershipAmount() }
+									<p>{ __( 'Or a new one.', 'jetpack' ) }</p>
```

Comment:
This feels a bit odd to me. I expected to read "Or create a new one" or "Or add a new one". But since others are okay with the copy, I'll approve, it may just be me. :)